.gitignore

```diff
@@ -160,4 +165,5 @@ cython_debug/
 #.idea/
 
 # Additional
-*.sqlite
\ No newline at end of file
+.vscode/
+.DS_Store
```

Comment:
does this catch all `DS_Store`s in all subfolders, too?

Reply:
I believe so!